snippets/csharp/VS_Snippets_Misc/wifcustomtokenst/cs/simplewebtoken/simplewebtoken.cs

```diff
@@ -28,18 +28,18 @@ public class SimpleWebToken : SecurityToken
 
         NameValueCollection _properties;
         //</Snippet3>
-        string _serilaizedToken;
+        string _serializedToken;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="SimpleWebToken"/> class.
-        /// This is internal contructor is only called from the <see cref="SimpleWebTokenHandler"/> when reading a token received from the wire.
+        /// This is internal constructor is only called from the <see cref="SimpleWebTokenHandler"/> when reading a token received from the wire.
         /// </summary>
         /// <param name="properties">The collection represents all the key value pairs in the token.</param>
```

Comment:
argh one more 😄  

```suggestion
         /// <param name="properties">The collection representing all the key value pairs in the token.</param>
```

Reply:
@mairaw Sure, fixed a couple of more typos, hope you don't mind.